test/TritonGPU/loop-pipeline-hip.mlir

```diff
@@ -263,3 +263,67 @@ module attributes {"triton_gpu.num-ctas" = 1 : i32, "triton_gpu.num-warps" = 4 :
     tt.return
   }
 }
+
+// -----
+
+// Check that the stream pipeliner updates atomic op in the k-loop correctly
+// CHECK-LABEL: _triton_gemm_kernel_atomic_rmw
+// CHECK: tt.atomic_rmw fadd, acq_rel, gpu
```

Comment:
Can you also insert `CHECK:` lines for the loop structure `scf.for` and `scf.yield` to be clear? Like L36 - L42.

Reply:
thanks for the comments, added.